lodepng.h

```diff
@@ -33,10 +33,10 @@ freely, subject to the following restrictions:
 #endif
 
 #if LV_PNG_USE_LV_FILESYSTEM
-#ifdef LV_LVGL_H_INCLUDE_SIMPLE
-#include <lvgl.h>
+#if defined(LV_LVGL_H_INCLUDE_SIMPLE)
```

Comment:
Why `#if defined` is better than `#ifdef` here?

Reply:
> Why `#if defined` is better than `#ifdef` here?

I refer to the wording here [lv_demo.h](https://github.com/lvgl/lv_demos/blob/master/lv_demo.h)

Reply:
Hmm, okay. It doesn't matter much so let's leave it as it is.